contracts/eden/src/atomicassets.cpp

```diff
@@ -51,4 +59,67 @@ namespace eden::atomicassets
       return result;
    }
 
+   void init_collection(eosio::name contract,
+                        eosio::name self,
+                        eosio::name collection,
+                        eosio::name schema_name)
+   {
+      ::atomicassets::collections_t collections(contract, contract.value);
+      if (auto pos = collections.find(collection.value); pos != collections.end())
+      {
+         eosio::check(pos->allow_notify, "Notifications are required");
+         eosio::check(std::find(pos->authorized_accounts.begin(), pos->authorized_accounts.end(),
+                                self) != pos->authorized_accounts.end(),
+                      "Contract is not authorized for the collection");
+         if (std::find(pos->notify_accounts.begin(), pos->notify_accounts.end(), self) ==
+             pos->notify_accounts.end())
+         {
+            // This will fail if the contract is not the author of the collection.
+            // In that case, it's the responsibility of the author to make sure
+            // that notifications are set up.
+            actions::addnotifyacc(contract, {self, "active"_n}).send(collection, self);
+         }
+      }
+      else
+      {
+         actions::createcol(contract, {self, "active"_n})
+             .send(self, collection, true, std::vector{self}, std::vector{self}, 0.05,
```

Comment:
can we please move this magic 0.05 to a proper constant? (I believe it's the market fees)